daemon/info.go

```diff
@@ -16,7 +16,7 @@ import (
 	"github.com/docker/docker/utils"
 )
 
-func (daemon *Daemon) SystemInfo() (*types.Info, error) {
+func (daemon *Daemon) SystemInfo(includeOperatingSystem bool) (*types.Info, error) {
```

Comment:
I'm not very fond of the Boolean parameter here; I think we use separate functions in other locations, e.g. https://github.com/docker/docker/blob/75f8bdd970e5355a6238e7b41a9f1aee9cd3f890/api/server/server_unix.go#L131
@tiborvass is that more what you were thinking?